pysat/tests/instrument_test_class.py

```diff
@@ -160,6 +160,19 @@ def test_instrument_test_dates(self, name):
             for tag in info[sat_id].keys():
                 assert isinstance(info[sat_id][tag], dt.datetime)
 
+    @pytest.mark.all_inst
```

Comment:
Suggest adding an additional test that goes something like:

```
@pytest.mark.all_inst
def test_instrument_attrs(self, name):
"""Check that module has required instrument attributes
"""
    req_attrs = ['acknowledgements', 'references', 'load', 'clean', 'download', 'init', 'platform', 'name', 'tag', 'sat_id']
    module = import_module(''.join(('.', name)), package=self.package.__name__)
    for sat_id in module.sat_ids.keys():
        for tag in module.sat_idspsat_id]:
            inst = pysat.Instrument(inst_module=module, tag=tag, sat_id=sat_id)
            for rattr in req_attrs:
                assert hasattr(inst, rattr)


```

May or may not be better to parametrize the list of attrs.  Also, made the required attribute list from memory.

Reply:
We have tests that check all these, but we may want to consolidate.  See
- test_modules_standard
- test_required_function_presence
- test_instrument_test_dates